docker/dockerfile

```diff
@@ -0,0 +1,31 @@
+# Use the rust image as the base image
+FROM rust:latest
+
+# Install the dependencies required by Rust-TEOS
+RUN apt-get update && \
+    apt-get -y --no-install-recommends install libffi-dev libssl-dev pkg-config libleveldb-dev && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Set the working directory to /srv
+WORKDIR /srv
```

Comment:
If we can find what the best practices are we can try to follow them. Otherwise, we may not need to create a dir for this (even though I don't think it matters much tbh).